### PR TITLE
Update /5/ url and Deleted double import of a component

### PR DESCRIPTION
**Pull request by yahya-cloud**
Updated the url link from https://training.plone.org/5/ to https://training.plone.org/ from and also deleted double import of single component

### CODE REVIEW

**Comment by stevepiercy** [comment]
We need to wait to a response to [@ksuess's question](https://github.com/plone/training/issues/672#issuecomment-1467426883) to determine the status of the Volto training and whether this is the best link to use going forward.

File: news/4512.documentation [hunk 1]
```diff
@@ -0,0 +1 @@
+Deleted second import and fixed training urls
```
Comment (stevepiercy) [inline news/4512.documentation:1]: Give yourself credit!

```suggestion
Deleted second import and fixed training URLs. @yahya-cloud

```

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Comment by yahya-cloud** [comment]
@stevepiercy if I'm correct this training is archived right and it'll be  https://training.plone.org/5/ to https://2022.training.plone.org/

File: packages/generator-volto/generators/app/templates/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ## Documentation
 
-A training on how to create your own website using Volto is available as part of the Plone training at [https://training.plone.org/5/volto/index.html](https://training.plone.org/5/volto/index.html).
+A training on how to create your own website using Volto is available as part of the Plone training at [https://2022.training.plone.org/volto/index.html](https://2022.training.plone.org/volto/index.html).
```
Comment (stevepiercy) [inline packages/generator-volto/generators/app/templates/README.md:3]: I apologize for the last-minute change of plans. We decided that the default Volto training should be the following.

```suggestion
[Volto Hands-On](https://training.plone.org/voltohandson/index.html) is a training on how to create your own website.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Comment by yahya-cloud** [comment]
thank you for help, I've made the requested changes can you please check

**Review by stevepiercy — APPROVED** [review]

**Comment by stevepiercy** [comment]
@yahya-cloud I think that you need to open a PR in the `plone/volto` repo, instead of from your fork, to run all the required CI builds. This PR cannot be merged until all the GitHub Actions pass.

**Comment by yahya-cloud** [comment]
thank you for your help this far, i have opened the new PR [#4523](https://github.com/plone/volto/pull/4523)

**Comment by davisagli** [comment]
Closing this PR since it was replaced by #4523

**Pull request closed without merging**